### PR TITLE
Fix commented out test

### PR DESCRIPTION
**Pull request by mladjan-gadzic**

### CODE REVIEW

File: trino-hms-hdfs-ranger/tests/trino/6_test_trino_no_drop_table_perm.sh [hunk 1]
```diff
@@ -12,8 +12,9 @@ echo "- INFO: Reusing policies from previous test"
...
 
-# We would expect this test to fail with the 'failMsg' but it fails with a diffrent error.
-# 'Query 20240308_161853_00006_6nb76 failed: The following metastore delete operations failed: drop table default.new_trino_test_table'
-failMsg="Permission denied: user [postgres] does not have [DROP] privilege on [default/$NEW_TRINO_TABLE_NAME]"
-
+# It is expected for this test to fail with the permission denied exception with message similar to 'Permission denied: user [postgres] does not have [DROP] privilege on [default/new_trino_test_table]', but it fails with a different error.
+# Thanks to xBis7 for investigation, it was found that permission denied exception is swallowed and a new exception is thrown with a different message.
+# This seems to done for better messaging experience, but it is misleading.
+# More info could be found on the link https://github.com/G-Research/gr-oss/issues/551#issuecomment-1994240651.
+failMsg="The following metastore delete operations failed: drop table default.new_trino_test_table"
```
Comment (xBis7) [inline trino-hms-hdfs-ranger/tests/trino/6_test_trino_no_drop_table_perm.sh:19]: Use `$NEW_TRINO_TABLE_NAME` for table name?

Comment (mladjan-gadzic) [reply trino-hms-hdfs-ranger/tests/trino/6_test_trino_no_drop_table_perm.sh]: Nice catch. Fixed.

**Comment by xBis7** [comment]
@mladjan-gadzic Have you run the test locally? Is it passing?

**Comment by mladjan-gadzic** [comment]
> @mladjan-gadzic Have you run the test locally? Is it passing?

I always run `test_all.sh` before submitting a PR. In this case I ran only this test and it passed.

**Review by xBis7 — APPROVED** [review]
LGTM! Thanks for the fix.

**Comment by mladjan-gadzic** [comment]
> LGTM! Thanks for the fix.

Thanks for the quick review! I am going to merge it.

**Pull request merged**